src/sentry/static/sentry/app/views/usageStats/usageStatsOrg.tsx

```diff
@@ -0,0 +1,248 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import capitalize from 'lodash/capitalize';
+import moment from 'moment';
+
+import AsyncComponent from 'app/components/asyncComponent';
+import Button from 'app/components/button';
+import ButtonBar from 'app/components/buttonBar';
+import Card from 'app/components/card';
+import {HeaderTitle} from 'app/components/charts/styles';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import {Panel, PanelBody} from 'app/components/panels';
+import QuestionTooltip from 'app/components/questionTooltip';
+import {t, tct} from 'app/locale';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+import {DataCategory, Organization} from 'app/types';
+
+import Chart from './chart';
+import {OrganizationUsageStats} from './types';
+import {formatUsageWithUnits} from './utils';
+
+type Props = {
+  organization: Organization;
+
+  dataCategory: DataCategory;
+  dataCategoryName: string;
+  onChangeDataCategory: (dataCategory: DataCategory) => void;
+} & AsyncComponent['props'];
+
+type State = {
+  orgStats: OrganizationUsageStats;
+} & AsyncComponent['state'];
+
+class UsageStatsOrganization extends AsyncComponent<Props, State> {
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    const {organization} = this.props;
+
+    // TODO(org-stats): Allow user to pick date range
+    const fourWeeksAgo = moment().subtract(31, 'days').unix();
```

Comment:
You could reuse the 'statsPeriod' logic we use in many other endpoints. That would give you more API consistency with the rest of the API and let you leverage shared components for choosing a date range.